Dockerfile.rocm

```diff
@@ -102,13 +102,19 @@ ENV RAY_EXPERIMENTAL_NOSET_ROCR_VISIBLE_DEVICES=1
 
 ENV VLLM_NCCL_SO_PATH=/opt/rocm/lib/librccl.so
 
+RUN yum -y remove python3-requests
+
+ENV VLLM_TARGET_DEVICE="rocm"
+ENV VLLM_PYTHON_EXECUTABLE="/usr/bin/python3"
+
+ENV PYTORCH_ROCM_ARCH=${FA_GFX_ARCHS}
+
 RUN --mount=type=cache,target=/root/.cache/pip \
     pip install -U -r requirements-rocm.txt \
-    && patch /opt/rocm/include/hip/amd_detail/amd_hip_bf16.h ./rocm_patch/rocm_bf16.patch \
     && python3 setup.py install \
-    && cp build/lib.linux-x86_64-cpython-39/vllm/_C.abi3.so vllm/ \
-    && cp build/lib.linux-x86_64-cpython-39/vllm/_punica_C.abi3.so vllm/ \
-    && cp build/lib.linux-x86_64-cpython-39/vllm/_moe_C.abi3.so vllm/ \
+    && cp build/lib.linux-x86_64-3.9/vllm/_C.abi3.so vllm/ \
```

Comment:
Might be worth it to directly `pip install .` instead of manually copying `.so`s